core/platform/taskqueue/cloud_tasks_emulator.py

```diff
@@ -35,8 +35,8 @@ class Task:
     the cloud tasks emulator.
     """
 
-    # In the type annotation below, payload is of type Dict[str, Any] because
-    # the payload here has no constraints.
+    # Here we use type Any because the payload can accept Dict and
+    # it has no constraints over it's values.
```

Comment:
it's --> its

But perhaps better: ...and this Dict has no constraints on its values.

(since it is not clear what "it" refers to in the original sentence).

Reply:
this Dict has no constraints on its values.

 added